lib/modules/versioning/hermit/index.spec.ts

```diff
@@ -0,0 +1,276 @@
+      ${'@1'}      | ${'@1.2'}    | ${false}
+      ${'@1.2'}    | ${'@1.2'}    | ${true}
+      ${'@1.2.3'}  | ${'@1.2'}    | ${false}
+      ${'@latest'} | ${'@1'}      | ${false}
+      ${'@stable'} | ${'@stable'} | ${true}
+    `('matches("$version") === $expected', ({ version, other, expected }) => {
+      expect(!!versioning.matches(version, other)).toBe(expected);
+    });
+  });
+
+  describe('isGreaterThan', () => {
+    test.each`
+      version      | other        | expected
+      ${'@1'}      | ${'@1.2'}    | ${true}
+      ${'@1.2'}    | ${'@1.2'}    | ${false}
+      ${'@1.2'}    | ${'@1.3'}    | ${false}
+      ${'@1.2.3'}  | ${'@1.2'}    | ${false}
+      ${'1.2.3'}   | ${'@latest'} | ${true}
+      ${'@latest'} | ${'@1'}      | ${false}
+      ${'@stable'} | ${'@latest'} | ${true}
+      ${'@latest'} | ${'@stable'} | ${false}
+    `(
+      'isGreaterThan("$version", "$other") === $expected',
+      ({ version, other, expected }) => {
+        expect(!!versioning.isGreaterThan(version, other)).toBe(expected);
+      }
+    );
+  });
+
+  describe('isLesserThanRange', () => {
+    test.each`
+      version      | other        | expected
+      ${'@1'}      | ${'@1.2'}    | ${false}
+      ${'@1.2'}    | ${'@1.2'}    | ${true}
+      ${'@1.2.3'}  | ${'@1.2'}    | ${true}
+      ${'@latest'} | ${'@1'}      | ${true}
+      ${'@stable'} | ${'@latest'} | ${false}
+      ${'@latest'} | ${'@stable'} | ${true}
+    `(
+      'isLessThanRange("$version") === $expected',
+      ({ version, other, expected }) => {
+        expect(!!versioning.isLessThanRange(version, other)).toBe(expected);
+      }
+    );
+  });
+
+  it('getSatisfyingVersion', () => {
+    expect(versioning.getSatisfyingVersion(['@1.1.1', '1.2.3'], '1.2.3')).toBe(
+      '1.2.3'
+    );
+    expect(
+      versioning.getSatisfyingVersion(
+        ['1.1.1', '@2.2.1', '2.2.2', '3.3.3'],
+        '2.2.2'
+      )
+    ).toBe('2.2.2');
+    expect(
+      versioning.getSatisfyingVersion(
+        ['1.1.1', '@1.3.3', '2.2.2', '3.3.3'],
+        '1.2.3'
+      )
+    ).toBeNull();
+  });
+
+  it('minSatisfyingVersion', () => {
+    expect(versioning.minSatisfyingVersion(['@1.1.1', '1.2.3'], '1.2.3')).toBe(
+      '1.2.3'
+    );
+    expect(
+      versioning.minSatisfyingVersion(
+        ['1.1.1', '@1.2.3', '2.2.2', '3.3.3'],
+        '2.2.2'
+      )
+    ).toBe('2.2.2');
+    expect(
+      versioning.minSatisfyingVersion(
+        ['1.1.1', '@1.2.2', '2.2.2', '3.3.3'],
+        '1.2.3'
+      )
+    ).toBeNull();
+  });
+
+  describe('sortVresions', () => {
+    it('sorts versions in an ascending order', () => {
+      expect(
+        [
+          '@1',
+          '1.1',
+          '1.2',
+          '1.2.3',
+          '1.3',
+          '@1.2',
+          '@2',
+          '2',
+          '2.1',
+          '@stable',
+          '@latest',
+        ].sort(versioning.sortVersions.bind(versioning))
+      ).toEqual([
+        '@latest',
+        '@stable',
```

Comment:
Do you have a reference for this sorting? I would expect `@latest` to be the highest version.